src/components/VideoRoom.js

```diff
@@ -658,32 +661,54 @@ class VideoRoom extends Component {
             })}>
                     <TopBar>
                         { remoteAudioMuted ?
-                            <IconButton edge='end' color='inherit' aria-label='Un-mmute Remote Audio'  onClick={this._toggleRemoteAudio.bind(this)}>
-                                <MicOffIcon />
-                            </IconButton>
+                            <Tooltip title="Un-mute Remote Audio">
+                                <IconButton edge='end' color='inherit' aria-label='Un-mmute Remote Audio'  onClick={this._toggleRemoteAudio.bind(this)}>
+                                    <MicOffIcon />
+                                </IconButton>
+                            </Tooltip>
                         :
-                            <IconButton edge='end' color='inherit' aria-label='Mute Remote Audio'  onClick={this._toggleRemoteAudio.bind(this)}>
-                                <MicIcon />
-                            </IconButton>
+                            <Tooltip title="Mute Remote Audio">
+                                <IconButton edge='end' color='inherit' aria-label='Mute Remote Audio'  onClick={this._toggleRemoteAudio.bind(this)}>
+                                    <MicIcon />
+                                </IconButton>
+                            </Tooltip>
                         }
                         { screensharing ?
-                            <IconButton edge='end' color='inherit' aria-label='Stop Screen Share'  onClick={this._stopScreenShare.bind(this)}>
-                                <StopScreenShareIcon />
-                            </IconButton>
+                            <Tooltip title="Stop Screen Share">
+                                <IconButton edge='end' color='inherit' aria-label='Stop Screen Share'  onClick={this._stopScreenShare.bind(this)}>
+                                    <StopScreenShareIcon />
+                                </IconButton>
+                            </Tooltip>
                         :
-                            <IconButton edge='end' color='inherit' aria-label='Screen Share'  onClick={this._startScreenShare.bind(this)}>
-                                <ScreenShareIcon />
+                            <Tooltip title="Start Screen Share">
+                                <IconButton edge='end' color='inherit' aria-label='Screen Share'  onClick={this._startScreenShare.bind(this)}>
+                                    <ScreenShareIcon />
+                                </IconButton>
+                            </Tooltip>
+                        }
+                        <Tooltip title="Open Transcription Window">
```

Comment:
I added in a recent PR a check on mqttUri being there in props to show this - and you should have a corresponding close transcription button too?

Reply:
Fixed